composer.json

```diff
@@ -11,13 +11,15 @@
     ],
     "require": {
         "php": "^5.6|^7.0",
-        "symfony/config": "^2.8|^3.0",
-        "symfony/dependency-injection": "^2.8|^3.0",
-        "symfony/filesystem": "^2.8|^3.0",
-        "symfony/http-kernel": "^2.8|^3.0",
-        "symfony/routing": "^2.8|^3.0"
+        "composer-plugin-api": "^1.1",
+        "symfony/config": "^3.0",
```

Comment:
Since removing `2.8` is a BC break anyways, why not make `3.3` the default version? Contao 4.4 is not compatible with Symfony `< 3.3`.

Reply:
removing 2.8 is not a BC break, it's just a removed dependency. I'm fine with 3.3 though, as Contao 4.3 is the only previously supported version and is outdated.